Misc/NEWS.d/next/Library/2023-02-21-14-57-34.gh-issue-102114.uUDQzb.rst

```diff
@@ -0,0 +1 @@
+Functions from :mod:`dis` module that accept source code string as argument now print shorter traceback if given string contains syntax/indentation error
```

Comment:
```suggestion
Functions in the :mod:`dis` module that accept a source code string as argument now
print a more concise traceback when the string contains a syntax or indentation error.
Patch by YOUR_NAME_HERE.
```

Reply:
That's just a suggestion. I can merge it as it is if you want.

Reply:
Don't forget to ``git pull`` because I merged main into this branch.

Reply:
Thank you. I removed the signature as I find these changes pretty minor :)